### DISCUSSION TITLE
social/me/

### DISCUSSION THREAD

**Discussion by giscus** [ThatCoders/thatcoders.github.io#9] [Comment]
# social/me/

鱼鱼塘博主  如果宇宙中真有什么终极的逻辑，那就是我们终有一天会在舰桥上重逢，直到生命终结。XAOXUU 目前是一个 iOS 开发者，代表作品有：ProHUD、ValueX 等。在业余时间也开发了 Stellar 博客主题，更多的作品可以去项目主页查看，希望大家喜欢～文章项目留言GitHub

https://thatcoder.gitee.io/social/me/

**Comment by tovarsh** [comment]
时间:2023.1.19 
社交about页面还没改
xaoxuu大佬看到别打我,在改了
